packages/grid/src/features/standard-table/components/StandardTableHeadRow.module.css

```diff
@@ -0,0 +1,7 @@
+.stickyHeader {
+    background-color: white;
+    position: sticky;
+    top: 0;
+    z-index: 1000;
```

Comment:
Can we allow custom z-index and top value?

Reply:
Ah, yes, top must be configurable, no? In case we stack sticky stuff.